motr/setup.c

```diff
@@ -1099,8 +1099,10 @@ static int cs_storage_init(const char *stob_type,
 /**
    Finalises storage for a request handler in a motr context.
  */
-static void cs_storage_fini(struct cs_stobs *stob)
+static void cs_storage_fini(struct m0_reqh_context *rctx)
```

Comment:
Why can't we free in cs_reqh_ctx_fini() as rc_addb_stlocation is member of rctx , instead of freeing in cs_storage_fini().